File: rust/src/models/boolean_resolved_entitlement_value.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct BooleanResolvedEntitlementValue {
    pub enabled: bool,
}

impl BooleanResolvedEntitlementValue {
    pub fn new(enabled: bool) -> Self {
        Self { enabled }
    }
}

File: typescript/src/models/booleanResolvedEntitlementValue.ts
// this file is @generated

export interface BooleanResolvedEntitlementValue {
  enabled: boolean;
}

export const BooleanResolvedEntitlementValueSerializer = {
  _fromJsonObject(object: any): BooleanResolvedEntitlementValue {
    return {
      enabled: object["enabled"],
    };
  },

  _toJsonObject(self: BooleanResolvedEntitlementValue): any {
    return {
      enabled: self.enabled,
    };
  },
};
